### PR TITLE
Add sqlalchemy-redshift recipe

### PR DESCRIPTION
**Pull request by mcg1969**
As with my watchtower script, it's not clear we can run the full test suite.

### CODE REVIEW

File: recipes/sqlalchemy-redshift/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{% set name = "sqlalchemy-redshift" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/sqlalchemy-redshift/sqlalchemy-redshift.git
+  git_rev: "{{ version }}"
```
Comment (scopatz) [inline recipes/sqlalchemy-redshift/meta.yaml]: Please do not use git for the package as it uses up extra CI time and resources.  Instead, grab the tarball from github. Thanks!

**Comment by scopatz** [comment]
@mcg1969 any movement on this?

**Comment by mcg1969** [comment]
@scopatz fixes made.

**Comment by scopatz** [comment]
Works locally, thanks!

**Pull request merged**